pkg/karmadactl/util/types.go

```diff
@@ -0,0 +1,23 @@
+package util
```

Comment:
I think we can rename the file to `commandgroup.go` or `command_group.go`.
`types.go` is usually used to define API types.